services/state/consensus/topic.proto

```diff
@@ -53,7 +53,7 @@ message Topic {
     /**
      * The topic's unique entity number in the Merkle state.
```

Comment:
`s/number/id/`
Note, this PR cannot merge before the services PR that mirrors it.

Reply:
this one https://github.com/hashgraph/hedera-services/pull/7216